src/opendr/perception/slam/full_map_posterior_gmapping/src/fmp_slam_eval/README.md

```diff
@@ -0,0 +1,356 @@
+# FMP SLAM Evaluation
+
+This module includes helper nodes and scripts, useful for characterizing and evaluating the results of some SLAM algorithms.
+# Nodes
+
+## err_collector
+
+This node collects the transtalional, rotational and total errors computed by another node and stores them in CSV files.
```

Comment:
Typo?
```suggestion
This node collects the translation, rotational and total errors computed by another node and stores them in CSV files.
```

Reply:
Unlike most packages in this repo probably, this is a traditional SLAM algorithm, so no Machine Learning involved :S

 * Submodule _openslam_gmapping_, and its wrapper _slam_gmapping_, are derived from the respective official ROS Perception packages, but were modified to support additional options and algorithms and are are thus not standard ROS anymore. The travis files come from the original ROS implementation.
 * Module _map_simulator_ is for generating test data and can definitely go under projects.
 * _FMP_SLAM_Evaluation_ basically ties everything together with launch files and extra scripts, but could go in projects as well.